app/main.py

```diff
@@ -1,3 +1,11 @@
 def cache(func):
-    # Write your code here
-    pass
+    ls = []
+
+    def wrapper(*args, **kwargs):
+        if args in ls:
+            print("Getting from cache")
+        else:
+            ls.append(args)
+            print("Calculating new result")
+        return func(*args, **kwargs)
```

Comment:
The function executes `func` for each call. The main idea is to somehow store the results for previous arguments and return them for calls with the same arguments without executing `func`.